plugins/modules/azure_rm_virtualmachine.py

```diff
@@ -949,6 +977,15 @@ def extract_names_from_blob_uri(blob_uri, storage_suffix):
     disable_password_authentication=dict(type='bool')
 )
 
+user_assigned_identities_spec = dict(
+    id=dict(type='list',default=[]),
+    append=dict(type='bool',default=True)
+)
+
+managed_identity_spec = dict(
+    type=dict(type='str',choices=['SystemAssigned','UserAssigned','SystemAssigned, UserAssigned','None'],required=True),
+    user_assigned_identities=dict(type='dict',options=user_assigned_identities_spec,default={}),
+)
```

Comment:
expected 2 blank lines!
```suggestion
)

```